### PR TITLE
add 2019.3.4-update

### PR DESCRIPTION
**Pull request by flowerornament**

### CODE REVIEW

**Comment by flowerornament** [comment]
@rmariani can you give this a quick read through?

**Comment by rmariani** [comment]
Sure

File: static-site/posts/updates/2019.3.4-update.udon [hunk 1]
```diff
@@ -0,0 +1,52 @@
...
+
+# ~2019.3.4 Update
+
+Here’s a short update from the team for the week of `~2019.3.4`:
+
+;img@"https://media.urbit.org/site/posts/updates/~2019.3.4-update-1.jpg"(width "100%");
+
+## Network
+
+We breached successfully. You can read about [what we fixed](https://github.com/urbit/arvo/pull/1101). Issues ocassionally crop up as people join the network, but it looks to be smooth sailing as we make our way toward [cc-release](https://github.com/urbit/urbit/tree/cc-release) and new ames.
```
Comment (rmariani) [inline static-site/posts/updates/2019.3.4-update.udon]: Our convention is to either capitalize Ames or write it as `%ames`. So just pick one of those and be consistent with it.

File: static-site/posts/updates/2019.3.4-update.udon [hunk 2]
```diff
@@ -0,0 +1,52 @@
...
+
+## @p Bug
```
Comment (rmariani) [inline static-site/posts/updates/2019.3.4-update.udon]: You should emphasize that this is purely cosmetic and does not affect Azimuth or contracts thereof

Comment (rmariani) [reply static-site/posts/updates/2019.3.4-update.udon]: Also instances of `@p` should be surrounded by tics

**Comment by flowerornament** [comment]
Thanks

File: static-site/posts/updates/2019.3.4-update.udon [hunk 3]
```diff
@@ -0,0 +1,52 @@
...
+```
+> `@p`3.108.299.008
+~morlyd-mogmev
+> `@p`479.733.505
+~morlyd-mogmev
+```
+
+We [announced the bug on Twitter](https://twitter.com/urbit/status/1105850450559918082) today, and [a conversation](https://github.com/urbit/arvo/issues/1105) is ongoing on Github.
+
+A number of solutions have been proposed – from fixing the algorithm and using a constraint solver to find a new ordering of phonemes such that all issued planets would keep their current names to hard-coding a new name for every affected planet – but none nearly as elegant as [this 3-line suggestion](https://github.com/urbit/arvo/issues/1105#issuecomment-472585937) from @max19, which elegantly corrects the implementation of the algorithm, preserving all existing planet names.
```
Comment (rmariani) [inline static-site/posts/updates/2019.3.4-update.udon]: This sentence is long and hard to follow

File: static-site/posts/updates/2019.3.4-update.udon [hunk 4]
```diff
@@ -0,0 +1,52 @@
...
+
+Stay tuned for a post with a full report.
+
+## Team
+
+*Jared* `~nidsut-tomdun` spent the week on the @p debacle and more than once generated all four billion planet names to check for collisions. *Ted* `~rovnys-ricfer` spent much of the week thinking about redesigning the PKI only to realize that the existing design is quite good. *Logan* `~tacryt-socryp` and *Isaac* `~novlud-padtyv` started working on the next major revision to Landscape. *Ben* `~pitmug-roptic` quietly [started working on](https://github.com/urbit/urbit/pull/1221) 64-bit Vere. `~pilfer-pandex` [brought the hammer down](https://github.com/urbit/docs/pull/399#issuecomment-472667240) on the pronunciation debate.
+
+Tlon is at the beginning a new batch of work for 2019, so new proposals are popping up almost daily. *Morgan* `~hidrel-fabtel` is working on revamping our proposal (“UP”) system so that the outside world can more easily participate in all of the work that’s going on. *Anthony* `~poldec-tonteg` and *Jared* `~nidsut-tomdun` are also working on standards for our repository documentation and open source development process.
```
Comment (rmariani) [inline static-site/posts/updates/2019.3.4-update.udon]: "work that's going on" would sound better as "work that's being done."

"open source development" should be "open-source development"

**Review by rmariani — CHANGES_REQUESTED** [review]
Ok, left some comments

File: static-site/posts/updates/2019.3.4-update.udon (hunk 4 above)
Comment (msutherl) [reply static-site/posts/updates/2019.3.4-update.udon]: prefer "going on" – incorporated the rest – thank you!

**Comment by flowerornament** [comment]
@rmariani Thanks! Prefer "going on" – incorporated the rest. See latest commit.

**Comment by rmariani** [comment]
LGTM
